test/login.test.js

```diff
@@ -42,42 +42,52 @@ describe('Test sidekick login', () => {
     nock.done();
   });
 
-  it('Opens login window and logs in via auth-cookie', async () => {
+  it('Opens login window and logs in cookie-less', async () => {
```

Comment:
Not fully sure what if there is enough value remaining in this test after the changes, because there might be too much mock behaviour.
Looking at the way tests are being executed, I'm not sure how else it could be done though... To have a fully meaningful test, it would need for the extension to be fully loaded including the background thread, so it can receive the message sent by the successful page login and then inject it via `modifyHeaders` rules into outgoing requests.